pages/legacy/module_edit.py

```diff
@@ -39,6 +45,14 @@ def title_header(self):
     def title(self):
         return self._title_regex.match(self.title_header.text).group(1)
 
+    @property
+    def portal_msg(self):
+        msg = WebDriverWait(self.driver, self.timeout).until(
+            EC.presence_of_element_located((By.CLASS_NAME, 'portalMessage'))
+        )
```

Comment:
I would say instead of using the low-level `WebDriverWait` class you should define a method that uses `self.is_element_present()` to check for the `portalMessage`, then use this form: `self.wait.until(lambda _: self.is_portal_message_present)`